modules/directives/src/main/scala/scala/build/preprocessing/directives/ScriptWrapper.scala

```diff
@@ -0,0 +1,37 @@
+package scala.build.preprocessing.directives
+
+import scala.build.directives.*
+import scala.build.errors.BuildException
+import scala.build.options.{BuildOptions, ScriptOptions}
+import scala.cli.commands.SpecificationLevel
+
+@DirectiveExamples("//> using objectWrapper")
+@DirectiveExamples("//> using delayedInitWrapper")
+@DirectiveUsage(
+  "//> using delayedInitWrapper",
+  """`//> using objectWrapper`
+    |
+    |`//> using delayedInitWrapper`
+    |""".stripMargin
+)
+@DirectiveLevel(SpecificationLevel.RESTRICTED)
+@DirectiveDescription("Set parameter for the code wrapper for scripts")
+final case class ScriptWrapper(
+  @DirectiveName("object.wrapper")
```

Comment:
IMO `wrapper.*` should be the prefix here
```suggestion
  @DirectiveName("wrapper.object")
```

Reply:
I also think so, but introducing this change now will break compatibility unfortunately :(